tests/test_examples.py

```diff
@@ -5,7 +5,17 @@
 def test_multiprocess_redis_database(pytester):
     pytester.copy_example()
 
-    # The `-n 2` are here is tightly coupled with the implementation of `test_split.py`.
+    # The `-n 4` are here is tightly coupled with the implementation of `test_split.py`.
```

Comment:
you could rm this comment and pass in 4 as a value

Reply:
to get the point across

Reply:
i'm not sure what you mean per se. Specifically this test wont work if you changed `-n 4` below to 3 or 5, because the number of tests in `test_split.py` is exactly 4. and I can't otherwise figure how to make that not true because the tests are static.

Reply:
```suggestion
# The number of splits here is tightly coupled with the implementation of `test_split.py`
split = 4  # or processes?
args = ["-vv", "-n", str(split), "test_split.py"]
result = pytester.inline_run(*args)
result.assertoutcome(passed=split, skipped=0, failed=0)
```

is what i mean

which couples it programmatically - so the comment can't become outdated